data/hai/hai missions.txt

```diff
@@ -1656,3 +1656,157 @@ mission "Strider notification"
+				`	"What would you want to do on <planet>? Isn't that an abandoned station?"`
+					goto question
+					to display
+						has "visited planet: Darkrest"
+				`	"Why do you want to go to <planet>?"`
+					goto "question 1"
+					to display
+						not "visited planet: Darkrest"
+				`	"Sure, I can take you there and back."`
+					goto accept
+				`	"How much can you pay me?"`
+				`	"Sorry, I'm busy right now. Go ask some other captain."`
+					decline
+			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you!" The Hai pauses to glance at his friends and, in almost a whisper, says, "Is <payment> okay?"`
+			choice
+				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
+					to display
+						has "visited planet: Darkrest"
+				`	"Why do you want to go to <planet>?"`
+					goto "question 1"
+					to display
+						not "visited planet: Darkrest"
+				`	"Sure, I'm not hurting for cash right now."`
+					goto accept
+				`	"No way, that's too low. Find someone else."`
+					decline
+			label question
+			`	"Well, that's kind of why we want to go there," says another one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
+				goto continue
+			label "question 1"
+			`	"Some guy in our dorm told us that there's ghosts haunting <planet>," says another one of the Hai, "and we wanted to call his bluff."`
+			label continue
+			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`
+			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`
+			choice
+				`	"Is this legal?"`
+				`	"Not my business where you spend your vacations. Come aboard."`
+					goto accept
+			`	The students look at each other, considering this question for what is apparently the first time. "We've heard of other people going there with no problems, as long as they were... discreet about it," says one of the students, trying to assure both you and their fellow classmates. Another mentions having heard something about needing a permit to enter, but he is quickly shushed by the rest of the group.`
+			choice
+				`	"I'm not about to risk getting in trouble for this. Find someone else."`
+					decline
+				`	"If anyone asks, I'll say I didn't know a permit was needed. Welcome aboard."`
+			label accept
+			`	The students race onboard your ship. As they settle into their bunks, you can hear them speculating about what they will find on <planet>.`
+				accept
+	on visit
+		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
+	on complete
+		payment 8000
```

Comment:
Move this to the on offer of the next mission so that the text replacement in its conversation works.
```suggestion
```